build/appveyor/MSVC-appveyor-build.bat

```diff
@@ -39,7 +39,7 @@ CD "%BUILDDIR%"                             || EXIT /B
     -DWITH_PYTHON=%WITH_PYTHON% ^
     -DWITH_%THREADMODEL%THREADS=ON ^
     -DWITH_SHARED_LIB=OFF ^
-    -DWITH_STATIC_LIB=ON                    || EXIT /B
+    -DWITH_STATIC_LIB=ON ^                    || EXIT /B
```

Comment:
The carat addition looks wrong to me; it belongs on the end of a string, so in this case I think it does nothing at all?

Reply:
Yes, you're right. I've removed it now.